submissions/onyinyeChukwuka/css/sign-up.css

```diff
@@ -0,0 +1,264 @@
+    font-size: 18px;
+    font-weight: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+    color: #5f5f5f;
+    margin-top: 40px;
+}
+
+.form {
+    border-radius: 23px;
+    border: solid 1px #707070;
+    background-color: #ffffff;
+    margin-top: 46px;
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-bottom: 20px;
+}
+
+.presentation {
+    width: 90%;
+    margin: auto;
+    margin-bottom: 180px;
+}
+
+.sidebar {
+    flex: 1;
+    margin-right: 10px;
+}
+
+#firstname, #lastname, #othernames, #phonenumber, #email, #password, #cpassword {
+    /* width: 521px; */
+    width: 100%;
+    height: 50px;
+    border-radius: 10px;
+    border: solid 2px #5f5f5f;
+    margin-bottom: 5px;
+}
+
+#terms {
+    font-family: Montserrat;
+    font-size: 40px;
+    font-weight: bold;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+    color: #0066f5;
+}
+
+#signupbutton {
+    width: 196px;
+    height: 48px;
+    border-radius: 4px;
+    background-color: #0066f5;
+    font-family: Raleway;
+    font-size: 18px;
+    font-weight: bold;
+    line-height: 0.67;
+    letter-spacing: normal;
+    text-align: center;
+    color: #ffffff;
+}
+
+.confirm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#submitbutton {
+    width: 108px;
+    height: 44px;
+    border: solid 1px #707070;
+    background-color: #ffffff;
+    font-family: Montserrat;
+    font-size: 18px;
+    font-weight: bold;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: center;
+    color: #0066f5;
+}
+
+/* for the footer */
+
+.footer {
+    background-color: #0066f5;
+    padding: 20px;
+    color: #fcf8f8;
+}
+
+
+.about {
+    font-family: Montserrat;
+    font-size: 18px;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+    color: #fcf8f8;
+}
+
+.images1 {
+    justify-content: space-around;
+    margin-left: 50px;
+}
+
+#newslettter {
+    width: 253px;
+    height: 44px;
+    border: solid 1px #707070;
+    background-color: #ffffff;
+}
+
+.newsletter ::placeholder {
+    font-family: Montserrat;
+    font-size: 16px;
+    font-weight: normal;
+    line-height: 1.19;
+    letter-spacing: normal;
+    color: #0066f5;
+    padding-left: 5px;
+}
+
+.newsletter h4 {
+    font-family: Montserrat;
+    font-size: 18px;
+    font-weight: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+    color: #fcf8f8;
+}
+
+.connect p {
+    font-family: Montserrat;
+    font-size: 18px;
+    font-weight: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+    color: #fcf8f8;
+    padding-left: 50px;
+}
+
+.footer>p {
+    text-align: center;
+}
+.newsletter-input {
+    display: flex;
+}
+
+@media screen and (min-width:768px)  {
+    .header {
+
+        height: 70px;
+    }
+
+    #firstname, #lastname, #othernames, #phonenumber, #email, #password, #cpassword {
```

Comment:
Inside of listing out all the `ids`. Kindly give this a `class` name. And use that `class` on all the HTML elements you want this style to have

Reply:
Inside of listing out all the `ids`. Kindly give this a `class` name. And use that `class` on all the HTML elements you want this style to have

Reply:
@Onyinye101 Kindly resolve this